### DISCUSSION TITLE
Request adding ZMK support for kpukboard

### DISCUSSION THREAD

**Discussion by guilhermeotsu** [manna-harbour/miryoku#219] [Q&A] [answered]
Here is ZMK shield for [kpukboard](https://github.com/kriku/kpukboard-zmk/tree/main/config/boards/shields/kpukboard). Thanks! 🙏

**Comment by manna-harbour** [comment, marked as answer]
Added in https://github.com/manna-harbour/miryoku_zmk/commit/d0c749b53d90d44b05a9ac442a690125b3a0f009.

Here's a sample build run: https://github.com/manna-harbour/miryoku_zmk/actions/runs/5220096497.

**Reply by guilhermeotsu** [reply]
Appreciate! One question, it's possible remap the extra pinky keys?

**Reply by manna-harbour** [reply]
Yes, see the _Mapping_ section at https://github.com/manna-harbour/miryoku/discussions/85.
